pkg/api/aim/api/request/run.go

```diff
@@ -22,6 +22,18 @@ type GetRunMetricsRequest []struct {
 	Context map[string]string `json:"context"`
 }
 
+// CreateRunArtifactRequest is a request object for `POST /runs/:id/artifacts` endpoint.
+type CreateRunArtifactRequest struct {
+	Iter    int64  `json:"iter"`
+	Step    int64  `json:"step"`
+	Caption string `json:"caption"`
+	Index   int64  `json:"index"`
+	Width   int64  `json:"width"`
+	Height  int64  `json:"height"`
+	Format  string `json:"format"`
+	BlobURI string `json:"blob_uri"`
```

Comment:
i'm wondering if `BlobURI` might better be the `StorageKey` or something (full path to append to the artifact uri). That way it's relying on the artifact storage configuration for the scheme/host and this would just be the key therein. ?

Reply:
though I do see that Aim refers to it as BlobURI `self.assertIn('blob_uri',` img)` so i can be convinced either way